signature/Cargo.toml

```diff
@@ -1,17 +1,16 @@
 [package]
 name = "signature"
-description = "Traits for cryptographic signature algorithms (e.g. ECDSA, Ed25519)"
 version = "2.3.0-pre.6"
 authors = ["RustCrypto Developers"]
-license = "Apache-2.0 OR MIT"
+edition = "2024"
+rust-version = "1.85"
 documentation = "https://docs.rs/signature"
-homepage = "https://github.com/RustCrypto/traits/tree/master/signature"
```

Comment:
Note that I removed the `homepage` field since it's not present in the most crates. I think we either should add it for all crates or remove them.

Personally, I don't think it's a correct use of the field. It does not provide any useful information to users, especially considering that README gets rendered on crates.io. I think we should use this field for a relevant section of  the RustCrypto book.